ozaria/site/components/teacher-dashboard/common/CodeDiff.vue

```diff
@@ -0,0 +1,96 @@
+<script>
+  const ace = require('lib/aceContainer')
```

Comment:
where is this used?

Reply:
oh where is my prev comment ? 
actually i just remember acediff requires an `ace` variable in scope so i keep it exists. doesn't manually use anywhere

Reply:
haha I would suggest adding a comment if its actually needed. Otherwise my IDE will keep asking me to delete this variable.

Though I find this weird. Maybe it's needed under window.ace? I don't know how it will access it otherwise